src/beeware_docs_tools/build_md_translations.py

```diff
@@ -28,6 +28,27 @@ def parse_args() -> Namespace:
     return args
 
 
+def config_update(update_version=True, language_directory=None, config_directory=None):
+    config_file = yaml.load(
+        open(SOURCE_DIR / "docs" / "config.yml"), Loader=yaml.SafeLoader
+    )
+    if update_version:
+        try:
+            version = metadata(config_file["extra"]["package_name"])["version"]
+            config_file["extra"]["version"] = version
+        except KeyError:
+            pass
+    else:
+        base_path = (language_directory / "shared_content").resolve()
+        config_file["markdown_extensions"]["pymdownx.snippets"]["base_path"] = [
+            "docs",
+            f"{base_path}",
+        ]
+
+    with (config_directory / "config.yml").open("w", encoding="utf-8") as config_temp:
+        yaml.dump(config_file, config_temp)
```

Comment:
This is likely really badly written code. I'm sure there's a better way to do it, but it's outside my abilities without duplicating a lot of code. Please let me know how it should be written, and I can update it and the places where it's called.